azure_monitor/src/azure_monitor/auto_collection/__init__.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+from opentelemetry.metrics import LabelSet, Meter
+
+from azure_monitor.auto_collection.dependency_metrics import DependencyMetrics
+from azure_monitor.auto_collection.performance_metrics import (
+    PerformanceMetrics,
+)
+from azure_monitor.auto_collection.request_metrics import RequestMetrics
+from azure_monitor.utils import PeriodicTask
+
+__all__ = [
+    "AutoCollection",
+    "DependencyMetrics",
+    "RequestMetrics",
+    "PerformanceMetrics",
+]
+
+
+class AutoCollection:
```

Comment:
I don't see `AutoCollection` being used anywhere? I believe we want this to be done automatically when a user creates an exporter.

Reply:
Currently user need to instantiate it, once wrapper is ready we can do it depending on configuration